docs/explanation/selecting-populations-for-study.md

```diff
@@ -0,0 +1,29 @@
+This page is aimed at researchers working with NHS England patient record data
+as provided by the OpenSAFELY backends.
+
+## Continuity of patient data
+
+Within the NHS in England, it is usually assumed that
+a patient's current primary care record is complete:
+when a patient moves practice, their record moves with them.
+
+However, a given NHS data set may only cover a subset of patients in England:
```

Comment:
thought: The use of "data set" here has the potential for confusion with "dataset" as used in ehrQL, but I'm not sure if we have some other term for this.

Reply:
The docs talk about data sources.